cmd/root.go

```diff
@@ -74,3 +82,17 @@ func initConfig() {
 		log.Info("config file not found, proceeding without it")
 	}
 }
+
+// isHelpCMD is used to determine if the user is trying to execute a help or a -h command,
+// if so then preflight does not need to load configurations or create a log file
+func isHelpCMD(args []string) bool {
+	var isHelpCMD bool
+	for _, a := range args {
+		if strings.Contains(a, help) || strings.Contains(a, dashH) {
+			isHelpCMD = true
+			break
+		}
+	}
```

Comment:
I think it would be great to refactor this method as a utility function for the future use. I suggest a method with this signature: `func ContainsAtLeast(keys []string, objects []string) bool` or `func ContainsAtLeast(keys ...string, objects []string) bool`

Reply:
The implementation was changed to use `PreRun` in the subcommands, so this method no longer exists. I'll keep things like this in mind in the future, to make things more generic and reusable.